src/headers/tomcrypt_cipher.h

```diff
@@ -1031,6 +1031,21 @@ int salsa20_test(void);
 
 #endif /* LTC_SALSA20 */
 
+#ifdef LTC_XSALSA20
+
+int xsalsa20_setup(salsa20_state *st, const unsigned char *key,   unsigned long keylen,
+                                      const unsigned char *nonce, unsigned long noncelen,
+                                      int rounds, unsigned char *subkey);
+int xsalsa20_test(void);
+
+#define xsalsa20_state     salsa20_state
+#define xsalsa20_ivctr64   salsa20_ivctr64
```

Comment:
Do we really have to provide this?
IIUC you could use it, but it's dangerous and therefor: will it even be used?

Reply:
removed